src/frontc/cabs2cil.ml

```diff
@@ -5804,6 +5798,7 @@ and doAliasFun vtype (thisname:string) (othername:string)
 and doDecl (isglobal: bool) : A.definition -> chunk = function
   | A.DECDEF ((s, nl), loc) ->
       currentLoc := convLoc(loc);
+      currentExpLoc := convLoc loc; (* eloc for local initializer assignment instruction *)
```

Comment:
Having the same thing right below each other one time in `(...)` and one time without it is a bit ugly.
I'd suggest to remove the parens in the line above.